03-transactions.md

```diff
@@ -827,6 +836,102 @@ at each bucket is a prefix of the desired index.
 
 # Appendix A: Expected Weights
 
+## Expected Weight of the Funding Transaction (v2 Channel Establishment)
+
+The *expected weight* of a funding transaction is calculated as follows:
+
+      inputs: 40 bytes + var_int + `scriptlen`
+        - previous_out_point: 36 bytes
+          - hash: 32 bytes
+          - index: 4 bytes
+        - var_int: ? bytes (dependent on `scriptlen`)
+        - script_sig: `scriptlen`
+        - witness <---- Cost for "witness" data calculated separately.
+        - sequence: 4 bytes
+
+      non_funding_outputs: 8 bytes + var_int + `scriptlen`
+        - value: 8 bytes
+        - var_int: ? bytes (dependent on `scriptlen`)
+        - script: `scriptlen`
```

Comment:
the largest standard output script, a bare multisig with 3 uncompressed keys, is `1 + 3 * (1 + 65) + 1 + 1 = 201`. this is possible with a 1 byte var_int

tx_add_output allows non standard scripts (line +264 in this pr, bolt 02), but i think this should be limited anyway to prevent DoS with really big scripts, or disallow non-standard scripts entirely

```suggestion
      non_funding_outputs: 9 bytes + `scriptlen`
        - value: 8 bytes
        - var_int: 1 byte
        - script: `scriptlen`
```